Pinta.Tools/Editable/EditEngines/BaseEditEngine.cs

```diff
@@ -489,6 +489,13 @@ public virtual bool HandleKeyDown (Document document, ToolKeyEventArgs e)
 			case Gdk.Key.Right:
 				HandleRight (e);
 				return true;
+			case Gdk.Key.bracketleft:
+				if (BrushWidth > 1)
```

Comment:
I'm not sure if the `BrushWidth > 1` check is necessary, since the widget seems to clamp against the min/max limits when setting the `Value` property?
If it is necessary, increasing the value would also need to clamp against the max value